test/lib/absinthe/phase/document/arguments/normalize_test.exs

```diff
@@ -43,6 +55,19 @@ defmodule Absinthe.Phase.Document.Arguments.NormalizeTest do
     }
   """
 
+  @fragment_query """
+    query Things($id: ID!) {
+      things {
+        ... thingsFragment
+      }
+    }
+    fragment thingsFragment on Things {
+      items(id: $id) {
+        id
+      }
+    }
+  """
```

Comment:
I'm curious how this tests illustrates the issue, because the operation is before the fragments here.